charts/yawol-controller/values.yaml

```diff
@@ -17,13 +17,11 @@ yawolCloudController:
   clusterRoleEnabled: true
   image:
     repository: ghcr.io/stackitcloud/yawol/yawol-cloud-controller
-    tag: latest
```

Comment:
I think we should just comment this lines out, to see that this values can be set.